cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers', 'uses_cloud_providers', 'setup_provider'),
+    pytest.mark.tier(2),
```

Comment:
@john-dupuy 
Just a question:
- Do we need ```uses_infra_providers``` and ```uses_cloud_providers``` separately? Because as per my understanding, ```setup_provider``` is applicable for all the providers.

Reply:
I don't think so, I was copying this context from another test. Thanks for pointing this out.